tide-whisperer.go

```diff
@@ -328,6 +334,8 @@ func main() {
 			}
 
 			if len(results) > 0 {
+				deviceNameTransformer.Transform(results)
```

Comment:
Should this call be included in the Dataset API endpoint as well?

https://github.com/tidepool-org/platform/blob/master/data/service/api/v1/users_datasets_get.go#L14

Reply:
@clintonium-119 does Tidepool Web use the platform endpoint to display devices?